internal/api/handlers.go

```diff
@@ -22,11 +23,15 @@ func NewGoCertRouter(env *Environment) http.Handler {
 	router.HandleFunc("POST /certificate_requests/{id}/certificate/reject", RejectCertificate(env))
 	router.HandleFunc("DELETE /certificate_requests/{id}/certificate", DeleteCertificate(env))
 
-	v1 := http.NewServeMux()
-	v1.HandleFunc("GET /status", HealthCheck)
-	v1.Handle("/api/v1/", http.StripPrefix("/api/v1", router))
+	monitoringMux := http.NewServeMux()
+	monitoringMux.HandleFunc("/status", HealthCheck)
+	monitoringMux.Handle("/metrics", metricsHandler)
 
-	return logging(v1)
+	api := http.NewServeMux()
+	api.Handle("/api/v1/", http.StripPrefix("/api/v1", router))
+	api.Handle("/", monitoringMux)
```

Comment:
Do we need a new mux for minitoring, here we could simply call 

```
router.Handle("/metrics", NewMetricsHandler())
```

Reply:
Done